esmvaltool/utils/cmorizers/obs/utilities.py

```diff
@@ -120,6 +120,8 @@ def _fix_coords(cube):
                 _fix_bounds(cube, cube.coord('longitude'))
                 cube.attributes['geospatial_lon_min'] = 0.
                 cube.attributes['geospatial_lon_max'] = 360.
+                nlon = len(cube.coord('longitude').points)
+                _roll_cube_data(cube, int(nlon/2) , -1)
```

Comment:
@tomaslovato can you pls remove the rolling from the `cmorize_obs_WOA` in this case?

Reply:
Done!